### PR TITLE
[v6r20] add performance tests

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES

*tests
NEW: new performance based on multi-mechanize tests are added 
ENDRELEASENOTES

### CODE REVIEW

File: tests/Performance/webserver/test_scripts/web.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class Transaction(object):
+
+  def __init__(self):
+    self.custom_timers = {}
+    self.conn = httplib.HTTPConnection("lhcb-cert-dirac.cern.ch")
+
+  def run(self):
+    # print len(datasets)
+    start_time = time.time()
+    self.conn.request("GET", "/DIRAC/s:LHCb-Certification/g:lhcb_prmgr/ExampleApp/getSelectionData")
```
Comment (fstagni) [inline tests/Performance/webserver/test_scripts/web.py]: That should be a parameter.

File: tests/Performance/webserver/test_scripts/web.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+"""
+It is used to test the tornado web framework
```
Comment (fstagni) [inline tests/Performance/webserver/test_scripts/web.py]: You should add some more info here, specifically how to use this test (how to invoke it) and what is supposed to be running (and when the test is considered a success and when not)

Comment (zmathe) [reply tests/Performance/webserver/test_scripts/web.py]: We have not discussed how to invoke the tests. We need to have some document where we describe what to do for doing performance tests. 
The following command invokes the test: 
multimech-run webserver

File: tests/Performance/webserver/test_scripts/web.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import time
+import httplib
+
+
+class Transaction(object):
+
+  def __init__(self):
+    self.custom_timers = {}
+    self.url = "/DIRAC/s:LHCb-Certification/g:lhcb_prmgr/ExampleApp/getSelectionData"
+    self.conn = httplib.HTTPConnection("lhcb-cert-dirac.cern.ch")
```
Comment (fstagni) [inline tests/Performance/webserver/test_scripts/web.py]: These 2 lines should be somehow configurable.

**Comment by fstagni** [comment]
BTW: the travis failing can be ignored, it's corrected in https://github.com/DIRACGrid/DIRAC/pull/3626

**Comment by zmathe** [comment]
Okay. Thanks!

File: tests/Performance/webserver/test_scripts/web.py (hunk 3 above)
Comment (zmathe) [reply tests/Performance/webserver/test_scripts/web.py]: well. This always changes depending what we want to test. How to configure it is another question. We have to see how we setup the performance testing framework and then we can decide how to configure. Maybe use of file or CS or other...

**Pull request merged**